CONTRIBUTING.md

```diff
@@ -78,7 +78,7 @@ $ cd docs/<guide>/
 
 2. Run the build command:
 ```
-$ asciidoctor --attribute imagesdir=topics/images master.adoc
+$ asciidoctor --attribute imagesdir=../../images master.adoc
```

Comment:
Actually, this should work without specifying any path at all. I'll test and confirm.